### ISSUE TITLE
[Accessibility Improvement] Add :focus styles

### ISSUE THREAD

**Issue by vavroom** [helpmeabstract/helpmeabstract#51]
There is no focus style defined along with hover styles. This means that a sighted keyboard user tabbing through the active elements on the page (links, forms, etc) may lose track of where the focus is at.

Should ensure that every style declaration of :hover also gets applied to :focus.
